app/code/Magebit/Faq/Model/Question.php

```diff
@@ -0,0 +1,228 @@
+<?php
+
+/**
+ * Magebit_Faq
+ *
+ * @category     Magebit
+ * @package      Magebit_Faq
+ * @author       Artjoms Fomins <info@magebit.com>
+ * @copyright    Copyright (c) 2023 Magebit, Ltd.(https://www.magebit.com/)
+ */
+
+declare(strict_types=1);
+
+namespace Magebit\Faq\Model;
+
+use Magebit\Faq\Api\Data\QuestionInterface;
+use Magento\Framework\App\ObjectManager;
+use Magento\Framework\DataObject\IdentityInterface;
+use Magento\Framework\Model\AbstractModel;
+use Magento\Framework\Validation\ValidationException;
+use Magento\Framework\Validator\HTML\WYSIWYGValidatorInterface;
+use Magento\Framework\Model\Context;
+use Magento\Framework\Registry;
+use Magento\Framework\Model\ResourceModel\AbstractResource;
+use Magento\Framework\Data\Collection\AbstractDb;
+use Magento\Backend\Model\Validator\UrlKey\CompositeUrlKey;
+use Magento\Framework\Exception\LocalizedException;
+
+/**
+ * Question model
+ *
+ * @method Question setStoreId(int $storeId)
+ * @method int getStoreId()
+ */
+class Question extends AbstractModel implements QuestionInterface
+{
+    /**
+     * Question cache tag
+     */
+    public const CACHE_TAG = 'question_b';
+
+    /**#@+
+     * Question's statuses
+     */
+    public const STATUS_ENABLED = 1;
+    public const STATUS_DISABLED = 0;
+
+    /**
+     * @var string
+     */
+    protected $_cacheTag = self::CACHE_TAG;
+
+    /**
+     * Prefix of model events names
+     *
+     * @var string
+     */
+    protected $_eventPrefix = 'question_block';
+
+    /**
+     * @var WYSIWYGValidatorInterface
+     */
+    private $wysiwygValidator;
+
+    /**
+     * @var CompositeUrlKey
+     */
+    private $compositeUrlValidator;
```

Comment:
If you are using inline type declarations, you should use the same approach in all files